packages/bundle-source/test/test-sanity.js

```diff
@@ -55,7 +55,7 @@ test('nestedEvaluate', async t => {
 
   const err2 = bundle.makeError2('bar');
   t.assert(
-    err2.stack.indexOf('(/bundled-source/index.js:10:') >= 0,
+    err2.stack.indexOf('(/bundled-source/index.js:8:') >= 0,
     'bundled source is in second stack trace with correct line number',
   );
```

Comment:
This change is fine for this PR because it does not make anything worse. But the test itself is not portable and will not work under the default `lockdown` setting which suppresses the `errorInstance.stack` property. Nor will it necessarily work on conformant JS machines, since this property is
   * not specified by the std and not even implemented on some conformant implementations (Moddable), 
   * Which stack frames are included are unspecified even by the proposal to standardize error stacks.